### ISSUE TITLE
Docker image run error

### ISSUE THREAD

**Issue by Parul-mahajan** [github-copilot-resources/copilot-metrics-viewer#98]
Hi,

I am trying to run the docker image in 1.7v, here the building of images was successful but the running of image has following error - 
![image](https://github.com/user-attachments/assets/1fd4ba99-e2f1-4b64-8890-3aae29929d47)

Please help.

**Comment by karpikpl** [comment]
hey @Parul-mahajan  could you provide your docker build and docker run commands?

**Comment by karpikpl** [comment]
For me tag 1.7 doesn't even build so I would build main. 
```bash
copilot-metrics-viewer-main git:(v1.7) docker build -t metrics-viewer .
```
@martedesco can we push 1.8 tag?

**Comment by Parul-mahajan** [comment]
Hi @karpikpl I ran the same commands as per the documentation and I was able to solve this by updating my docker file with the following - ```
# Stage 1: Build the Vue.js application
FROM node:22 AS build-stage
WORKDIR /app
COPY package*.json ./
RUN npm install
COPY . .
RUN npm run build

# Stage 2: Serve the application with Nginx
FROM nginx:1.27 AS production-stage
RUN apt-get update && apt-get install -y dos2unix

COPY --from=build-stage /app/dist /usr/share/nginx/html
COPY --from=build-stage /app/dist/assets/app-config.js /usr/share/nginx/html-template/app-config.template.js
COPY ./docker-entrypoint.d/*.sh /docker-entrypoint.d/
RUN chmod +x /docker-entrypoint.d/*.sh
RUN chmod +x /docker-entrypoint.d/99-config-app.sh  
RUN dos2unix /docker-entrypoint.d/99-config-app.sh  
EXPOSE 80
CMD ["nginx", "-g", "daemon off;"]

```

**Comment by karpikpl** [comment]
I think that solved in main and will be available in 1.8 release, hence I'm closing the issue.
